src/core/parse.js

```diff
@@ -342,6 +354,36 @@ foam.CLASS({
 
 foam.CLASS({
   package: 'foam.parse.compiled',
+  name: 'Range',
+  extends: 'foam.parse.compiled.State',
+
+  properties: [
+    {
+      name: 'from',
+      final: true,
+    },
+    {
+      name: 'to',
+      final: true,
```

Comment:
Done. All jscs-backed style complaints cleaned up for files other than `core/parse.js`.